README*.md

```diff
@@ -1,3 +1,8 @@
+\* = `plotlyupdate.m` is currently disabled. Please download the latest version 
```

Comment:
Ah I see. Will README*.md still show up as the readme in the github repo? Or can we rename it to something else that GitHub will recognize?